### PR TITLE
Added Intermediate Step Screen

### PR DESCRIPTION
**Pull request by omartehsin1**
Part 2 - `Adds Intermediate Step Screen`

Follow up PRs to continue the rest of the full as outlined below.

## Full Description
10% of people who have entered a OTK have not uploaded their codes.

## Motivation
When someone has entered their OTK but has not uploaded their codes
They need to know they are not finished their task of letting people know they've been exposed
And they need to know what to do next
So that their close contacts will receive an exposure notification

## Acceptance criteria

**Main screen**

Given people have entered their OTK but have not uploaded their codes
When they leave the flow or otherwise land on the main page of the app
Then they see messaging that they are not quite done ([design](https://www.figma.com/file/0pLawavG1fnBTXSYn4pdKZ/Exposure-notification?node-id=14333%3A26))

**Menu**

Given people have entered their OTK but have not uploaded their codes
When they open the menu
Then they see the prompt to share exposures ([design](https://www.figma.com/file/0pLawavG1fnBTXSYn4pdKZ/Exposure-notification?node-id=14333%3A52))

**Intermediate instructions screen**

Given people have entered their OTK but have not uploaded their codes
When they click "Finish sharing exposures"
Then they see the intermediate screen with steps, and the first step is now "Done" ([design](https://www.figma.com/file/0pLawavG1fnBTXSYn4pdKZ/Exposure-notification?node-id=14333%3A740))
And when they click "Next"
Then they follow the rest of the existing process (entering date of symptom onset, etc)

**Confirmation of completed the flow**

When people have entered their OTK and shared their exposures, completing the flow
Then their screen says "Thanks for helping fight COVID-19" and tells them to share any new exposures going forward ([text change - design)](https://www.figma.com/file/0pLawavG1fnBTXSYn4pdKZ/Exposure-notification?node-id=14333%3A521)

**Sharing exposures on subsequent days**
Given someone has entered a OTK and shared exposures on a previous day
When they open their app
Then they see the "Share new exposures screen" ([text change - design](https://www.figma.com/file/0pLawavG1fnBTXSYn4pdKZ/Exposure-notification?node-id=14333%3A1112)).

### CODE REVIEW

File: ios/CovidShield.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -571,7 +571,7 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = ca.gc.hcsc.canada.stopcovid;
 				PRODUCT_NAME = CovidShield;
-				PROVISIONING_PROFILE_SPECIFIER = "Stop COVID Development Profile";
+				PROVISIONING_PROFILE_SPECIFIER = "Stop COVID Development";
```
Comment (timarney) [inline ios/CovidShield.xcodeproj/project.pbxproj:574]: Please revert this

Comment (omartehsin1) [reply ios/CovidShield.xcodeproj/project.pbxproj]: Whoops, don't know how that snuck in there. Reverted.

**Comment by timarney** [comment]
Did a quick check - looking good.

We'll get some test builds out for further QA + review.

**Comment by timarney** [comment]
@maryrae-qa can you give a thumbs up here based on QA?

**Comment by aklef** [comment]
> @aklef
> can you give a thumbs up here based on QA?

I have tested and reviewed acceptance criteria #4 and #5 and they are fine

**Comment by maryrae-qa** [comment]
PR Reviewed/Tested of Acceptance Criteria #1, 2 ad 3 has been completed and logged. iOS (Builds 194& 205) & Android (Builds 205 & 206). All tests passed.

**Review by timarney — APPROVED** [review]

**Pull request merged**